graphql_bookstore_service/README.md

```diff
@@ -0,0 +1,149 @@
+{allBooks {title}}
+```
+
+Response: 
+```json
+{
+ "data": {
+   "allBooks": [
+     { "title": "Pride and Prejudice" },
+     { "title": "Sense and Sensibility" },
+     { "title": "War and Peace" },
+     { "title": "Anna Karenina" }
+   ]
+ }
+}
+```
+CURL command  to request the same:
+curl -X POST -H "Content-type: application/json" -d '{ "query": "{allBooks {title}}" }' 'http://localhost:9090/graphql'
+
+## Sample Request 2:  Get more details of all books
+
+Users can request the exact information they need in the format they prefer without having different endpoints.
+
+GraphQL query : 
+```
+{allBooks {title, author, reviews{ratingsCount, averageRating}}}
+```
+
+Response :
+
+```json
+{
+  "data": {
+    "allBooks": [
+      {
+        "title": "Pride and Prejudice",
+        "author": "Jane Austen",
+        "reviews": {
+          "ratingsCount": 1,
+          "averageRating": 5
+        }
+      },
+      {
+        "title": "Sense and Sensibility",
+        "author": "Jane Austen",
+        "reviews": {
+          "ratingsCount": 3,
+          "averageRating": 4
+        }
+      },
+      {
+        "title": "War and Peace",
+        "author": "Leo Tolstoy",
+        "reviews": {
+          "ratingsCount": 5,
+          "averageRating": 4
+        }
+      },
+      {
+        "title": "Anna Karenina",
+        "author": "Leo Tolstoy",
+        "reviews": {
+          "ratingsCount": 1,
+          "averageRating": 4
+        }
+      }
+    ]
+  }
+}
+```
+
+CURL command to send the same request:
+
+```
+curl -X POST -H "Content-type: application/json" -d '{ "query": "{allBooks {title, author, reviews{ratingsCount, averageRating}}}" }' 'http://localhost:9090/graphql'
+```
+
+## Sample Request 3:  Get details of books with input parameter  
+
+GraphQL Query:  
+```
+{book(isbn: "9780099512240") {title, year}}
+```
+
+Response:
+
+```json
+{
+  "data": {
+    "book": {
+      "title": "War and Peace",
+      "year": 1867
+    }
+  }
+}
+```
+
+CURL Command to send the same request:
+```
+curl -X POST -H "Content-type: application/json" -d '{ "query": "{book(isbn: \"9780099512240\") {title, year}}" }' 'http://localhost:9090/graphql'
+```
+
+## Sample Request 4: Mutation to insert data into the database
+
+GraphQL Query:
+```
+mutation {addBook(isbn: "9781683836223", title: "Harry Potter", author: "J. K. Rowling", year: 2007)
+  {isbn}
+}
+```
+Response:
+
+```json
+{
+ "data": {
+   "addBook": 6
+ }
+}
+```
+
+CURL Command to send the same request:
+```
+curl -X POST -H "Content-type: application/json" -d '{ "query": "mutation {addBook(isbn: \"9781683836223\", title: \"Harry Potter\", author: \"J. K. Rowling\", year: 2007){isbn}}" }' 'http://localhost:9090/graphql'
+```
+
+You can find the original sample here: [https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-graphql-with-multiple-datasources](https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-graphql-with-multiple-datasources)
```

Comment:
IMO, this is not needed as it is like we are referring to personal repo from the official guides repo :)

Reply:
Updated the file :)